.github/workflows/main2.yml

```diff
@@ -0,0 +1,21 @@
+name: Cypress Unit Tests
```

Comment:
@sdevalapurkar Should this be renamed as 'Cypress Tests'? Just to avoid confusion as 'Jest' is used for unit tests?